patterns/1-initial/document-architecture-decisions.md

```diff
@@ -0,0 +1,230 @@
+## Title
+
+Document Architecture Decisions
+
+## Patlet
+
+InnerSource contributors often face challenges in grasping the system's design rationale, which can result in misalignment between maintainers, contributors, and stakeholders — potentially discouraging participation. To enhance decision-making and transparency, we recommend capturing architecture decisions and their consequences in a lightweight, accessible format to streamline onboarding, clarify decisions, and support long-term project sustainability.
```

Comment:
I feel we could make the 2nd sentence more clear.

Right now it is of the form:
`<intended benefit>, <approach>, <more intended benefits>`

However I cannot come up with a better version right now either, therefore let's leave it as is for now.

I am leaving this comment open.